inbm/dispatcher-agent/tests/unit/source/test_ubuntu_source_cmd.py

```diff
@@ -318,15 +318,15 @@ def test_success_add_gpg_key_method(self, mock_verify_source):
               patch("dispatcher.source.ubuntu_source_manager.add_gpg_key")):    
             command.add(parameters)
 
-    def test_raises_when_space_check_fails(self):
+    def test_raises_when_name_check_fails(self):
         parameters = ApplicationRemoveSourceParameters(
-            gpg_key_name="example_source.gpg", source_list_file_name="../example_source.list"
+            gpg_key_name="example_source.gpg", source_list_file_name="invalid/chars*in:name.list"
```

Comment:
**Suggestion:** Avoid using hardcoded strings for file names and use a constant instead. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            EXAMPLE_SOURCE_LIST = "example_source.list"
            parameters = ApplicationRemoveSourceParameters(
                gpg_key_name="example_source.gpg", source_list_file_name=EXAMPLE_SOURCE_LIST
            )
```

Reply:
A constant is used.